server.js

```diff
@@ -10,15 +10,26 @@ app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json({ limit: "10mb", extended: true }));
 
 app.post("/api/predict", function (req, res) {
-  const img = req.body.img;
+  const img = req.body.fcmResponse.img;
+  const secure = req.body.fcmResponse.secure;
+  const levelOfAssurance = req.body.levelOfAssurance;
+  const metadata = req.body.metadata;
+
+  const data = {
+    img: img,
+    secure: secure,
+    level_of_assurance: levelOfAssurance,
+    metadata: metadata,
+  };
 
   const request = new RequestBuilder()
     .withBaseUrl(process.env.BASE_URL)
     .withPemFilePath(process.env.PEM_FILE_PATH)
     .withEndpoint(process.env.ENDPOINT)
-    .withPayload(new Payload({ img }))
+    .withPayload(new Payload(data))
     .withMethod("POST")
     .withHeader("X-Yoti-Auth-Id", process.env.SDK_ID)
+    .withQueryParam("secure", !!secure)
```

Comment:
It will allow us to call:
- `?secure=true`
- `?secure=false`
But not the endpoint without "secure" in the query, no?

Reply:
Correct :+1:

Reply:
Ok, if we want to test without the query parameter we just need to delete the line
     .withQueryParam("secure", !!secure)
and recompile the package no?

Reply:
Yes, that would be possible.